pubspec.yaml

```diff
@@ -1,11 +1,10 @@
 name: scrolling_page_indicator
 description: Flutter implementation of page view indicator like Instagram
-version: 0.1.3
-author: "Nikita Zhelonkin <niks.mbox@gmail.com>"
```

Comment:
?

Reply:
`author` is deprecated, use verified publishers instead

https://stackoverflow.com/a/59942840